app/routers/event.py

```diff
@@ -66,63 +66,71 @@ async def create_new_event(request: Request, session=Depends(get_db)):
     category_id = data.get('category_id')
 
     invited_emails = get_invited_emails(data['invited'])
-    uninvited_contacts = get_uninvited_regular_emails(session, owner_id,
-                                                      title, invited_emails)
+    uninvited_contacts = get_uninvited_regular_emails(session, owner_id, title,
+                                                      invited_emails)
 
     if is_zoom:
         raise_if_zoom_link_invalid(location)
 
-    event = create_event(session, title, start, end, owner_id, content,
-                         location, invited_emails, category_id=category_id)
+    event = create_event(session,
+                         title,
+                         start,
+                         end,
+                         owner_id,
+                         content,
+                         location,
+                         invited_emails,
+                         category_id=category_id)
     message = ''
     if uninvited_contacts:
         message = f'Forgot to invite {", ".join(uninvited_contacts)} maybe?'
-    return RedirectResponse(router.url_path_for('eventview', event_id=event.id)
-                            + f'?{message}', status_code=status.HTTP_302_FOUND)
+    return RedirectResponse(
+        router.url_path_for('eventview', event_id=event.id) + f'?{message}',
+        status_code=status.HTTP_302_FOUND)
 
 
-@router.get("/{event_id}")
-async def eventview(request: Request, event_id: int,
+@router.get('/{event_id}')
+async def eventview(request: Request,
+                    event_id: int,
                     db: Session = Depends(get_db)):
     event = by_id(db, event_id)
     start_format = '%A, %d/%m/%Y %H:%M'
-    end_format = ('%H:%M' if event.start.date() == event.end.date()
-                  else start_format)
+    end_format = ('%H:%M'
```

Comment:
I think the prev format was more readable

Reply:
went back to original, sorry